inc/widgets.php

```diff
@@ -328,8 +343,11 @@ public function widget( $args, $instance ) {
 				?><a class="social-media-icon social-media-icon-<?php echo $id ?> social-media-icon-size-<?php echo esc_attr($instance['size']) ?>" href="<?php echo esc_url( $instance[$id], array('http', 'https', 'mailto', 'skype', 'callto', 'tel', 'sms') ) ?>" title="<?php echo esc_html( get_bloginfo('name') . ' ' . $name ) ?>" <?php if(!empty($instance['new_window'])) echo 'target="_blank"'; ?>><?php
 
 				$icon = apply_filters('vantage_social_widget_icon_'.$id, '');
-				if(!empty($icon)) echo $icon;
-				else echo '<span class="fa fa-' . $id . '"></span>';
+				if( ! empty( $icon ) ) {
```

Comment:
Couldn't this just be a ternary operator?

```suggestion
				echo !empty($icon) ? $icon : '<span class="fa fa-' . $id . '"></span>'
```

Reply:
I may have accidentally changed removed this in haste as it's really time-consuming test and I wanted to cut down on the number of times I had to run it. I was relying on a premium user's website to run this test as I can't seem to find an environment I control where the test doesn't timeout. I doubt they'll be keen for me to run this test again at this point.

@Misplon Are you able to try the suggested change to see if it flags this line?

Reply:
I think I got it to run on our demo server. Let's hit this change, I'll then build and test on the demo server. Thanks!